icicle/src/hash/keccak/keccak.cu

```diff
@@ -245,7 +245,7 @@ namespace keccak {
       CHK_IF_RETURN(cudaMallocAsync(&output_device, number_of_blocks * (D / 8), stream));
     }
 
-    int number_of_threads = 1024;
+    int number_of_threads = 512;
```

Comment:
Why is this being reduced?

Reply:
Keccak kernel requires a lot of registers, so running with 1024 threads won't fit the GPU

Reply:
Was this working before hand though?